### PR TITLE
implemented createModules, AssignModulesToStudents, AssignModulesToIn…

### PR DESCRIPTION
**Pull request by GarushEdvardovich**
…structors

### CODE REVIEW

File: LearningMissionSimulation/PlaygroundGarush.cs [hunk 1]
```diff
@@ -51,26 +59,86 @@ public void ActivateAccounts()
...
+                i++;
+            }
         }
 
         public void CreateModules(int moduleCount)
+        {
+            int i = 0;
+            while (i < moduleCount)
+            {
+                Guid subjectId = subjectIdList[AttributeGenerator.random.Next(0,subjectIdList.Count-1)];
```
Comment (learning-mission) [inline LearningMissionSimulation/PlaygroundGarush.cs:130]: Please use "Format Document" to fix formatting issues

**Review by learning-mission — CHANGES_REQUESTED** [review]
Please fix issues that I mentioned in the comments. Thanks.

File: LearningMissionSimulation/PlaygroundGarush.cs [hunk 2]
```diff
@@ -127,7 +125,7 @@ public void CreateModules(int moduleCount)
             int i = 0;
             while (i < moduleCount)
             {
-                Guid subjectId = subjectIdList[AttributeGenerator.random.Next(0,subjectIdList.Count-1)];
+                Guid subjectId = subjectIdList[AttributeGenerator.random.Next(0, subjectIdList.Count - 1)];
```
Comment (learning-mission) [inline LearningMissionSimulation/PlaygroundGarush.cs:128]: The upper limit is subjectIdList.Count, please fix.

File: LearningMissionSimulation/PlaygroundGarush.cs [hunk 3]
```diff
@@ -82,11 +82,11 @@ List<Module> GetModuleList()
...
             minModuleCount = Math.Min(allModuleCount, minModuleCount);
             maxModuleCount = Math.Min(allModuleCount, maxModuleCount);
-            int Count = AttributeGenerator.random.Next(minModuleCount, maxModuleCount);
+            int moduleCount = AttributeGenerator.random.Next(minModuleCount, maxModuleCount + 1);
 
-            for (int i = 0; i < Count; i++)
+            for (int i = 0; i < moduleCount; i++)
             {
-                Guid id = moduleIdList[AttributeGenerator.random.Next(0, moduleIdList.Count - 1)];
+                Guid id = moduleIdList[AttributeGenerator.random.Next(0, moduleIdList.Count)];
```
Comment (learning-mission) [inline LearningMissionSimulation/PlaygroundGarush.cs:89]: Your logic allows to add the same module multiple times. You should avoid that.

**Review by learning-mission — CHANGES_REQUESTED** [review]

File: LearningMissionSimulation/PlaygroundGarush.cs (hunk 3 above)
Comment (GarushEdvardovich) [reply LearningMissionSimulation/PlaygroundGarush.cs:89]: fixed

File: LearningMissionSimulation/PlaygroundGarush.cs [hunk 4]
```diff
@@ -51,26 +59,86 @@ public void ActivateAccounts()
...
+            int allModuleCount = moduleIdList.Count;
+            int minModuleCount = 2;
+            int maxModuleCount = 5;
+            minModuleCount = Math.Min(allModuleCount, minModuleCount);
+            maxModuleCount = Math.Min(allModuleCount, maxModuleCount);
+            int Count = AttributeGenerator.random.Next(minModuleCount, maxModuleCount);
+
+            for (int i = 0; i < Count; i++)
+            {
+                Guid id = moduleIdList[AttributeGenerator.random.Next(0, moduleIdList.Count - 1)];
```
[the remark this answers is not in the document]
Comment (GarushEdvardovich) [reply LearningMissionSimulation/PlaygroundGarush.cs]: Done

File: LearningMissionSimulation/PlaygroundGarush.cs [hunk 5]
```diff
@@ -51,26 +59,86 @@ public void ActivateAccounts()
...
+        List<Module> GetModuleList()
+        {
+
+            List<Module> moduleList = new List<Module>();
+            int allModuleCount = moduleIdList.Count;
+            int minModuleCount = 2;
+            int maxModuleCount = 5;
+            minModuleCount = Math.Min(allModuleCount, minModuleCount);
+            maxModuleCount = Math.Min(allModuleCount, maxModuleCount);
+            int Count = AttributeGenerator.random.Next(minModuleCount, maxModuleCount);
```
[the remark this answers is not in the document]
Comment (GarushEdvardovich) [reply LearningMissionSimulation/PlaygroundGarush.cs]: Done

File: LearningMissionSimulation/PlaygroundGarush.cs (hunk 4 above)
[the remark this answers is not in the document]
Comment (GarushEdvardovich) [reply LearningMissionSimulation/PlaygroundGarush.cs]: Done

File: LearningMissionSimulation/PlaygroundGarush.cs [hunk 6]
```diff
@@ -51,26 +59,86 @@ public void ActivateAccounts()
...
+            foreach (var instructor in instructorList)
+            {
+                instructor.ModuleList = GetModuleList();
+                Console.WriteLine(instructor);
+            }
+        }
+
+        List<Module> GetModuleList()
+        {
+
```
[the remark this answers is not in the document]
Comment (GarushEdvardovich) [reply LearningMissionSimulation/PlaygroundGarush.cs:76]: Done

**Pull request merged**